contributors/devel/release/issues.md

```diff
@@ -0,0 +1,193 @@
+# Targeting issues and PRs to release milestones
+
+This document describes how to target issues and PRs to a release.
+The process for shepherding issues into the release by the owner, release team, and GitHub bot
+is outlined below.
+
+## Definitions
+
+- *issue owners*: creator, assignees, and user who moved the issue it into a release milestone
+- *Y days*: refers to business days (using the location local to the release-manager M-F)
+- *post feature freeze* / *code slush*: starts when the master branch only accepts PRs for the release milestone
```

Comment:
Feature Freeze is an overloaded term.  I know PM also uses this to refer to when no additional issues/features are made available in the features repo.
https://github.com/kubernetes/features/blob/master/release-1.7/release-1.7.md